install/migrations/update_10.0.3_to_10.0.4/inventory.php

```diff
@@ -47,3 +47,6 @@
 
 //new right value for locked_field (previously based on config UPDATE)
 $migration->addRight('locked_field', CREATE | PURGE, ['config' => UPDATE]);
+
+//add remote_addr / last_boot
+$migration->addField(getTableForItemType(Agent::class), 'remote_addr', 'string');
```

Comment:
Hardcoded names have to be used in migrations, as they should not be affected by changes in naming rules.

```suggestion
$migration->addField('glpi_agents', 'remote_addr', 'string');
```